src/utils/countryCurrency/countryExchange.json

```diff
@@ -0,0 +1,673 @@
+        "value": 800
+    },  
+    {
+        "currencyCode": "ARS",
+        "value": 100
+    },
+    {
+        "currencyCode": "AUD",
+        "value": 1.6
+    },
+    {
+        "currencyCode": "AWG",
+        "value": 2.25
+    },
+    {
+        "currencyCode": "AZN",
+        "value": 2
+    },
+    {
+        "currencyCode": "BAM",
+        "value": 2
+    },
+    {
+        "currencyCode": "BBD",
+        "value": 2.459619
+    },
+    {
+        "currencyCode": "BDT",
+        "value": 100
+    },
+    {
+        "currencyCode": "BGN",
+        "value": 2
+    },
+    {
+        "currencyCode": "BHD",
+        "value": 0.5
+    },
+    {
+        "currencyCode": "BIF",
+        "value": 2370
+    },
+    {
+        "currencyCode": "BND",
+        "value": 1.6
+    },
+    {
+        "currencyCode": "BOB",
+        "value": 8.5
+    },
+    {
+        "currencyCode": "BRL",
+        "value": 6.5
+    },
+    {
+
+        "currencyCode": "BSD",
+        "value": 1.2
+    },
+    {
+        "currencyCode": "BTN", 
+        "value": 90       
+    },
+    {
+        "currencyCode": "BWP",
+        "value": 13
+    },
+    {
+        "currencyCode": "BYN",
+        "value": 3
+    },
+    {
+        "currencyCode": "BYR",
+        "value": 23800
+    },
+    {
+        "currencyCode": "BZD",
+        "value":2.45
+    },
+    {
+        "currencyCode": "CAD",
+        "value": 1.50
+    },
+    {
+        "currencyCode": "CDF",
+        "value": 2400
+    },
+    {
+        "currencyCode": "CHF",
+        "value": 1.0
+    },
+    {
+        "currencyCode": "CLF",
+        "value": 0.032
+    },
+    {
+        "currencyCode": "CLP",
+        "value": 880.25
+    },
+    {
+        "currencyCode": "CNY",
+        "value": 8.0
+    },
+    {
+        "currencyCode": "COP",
+        "value": 4220.0
+    },
+    {
+        "currencyCode": "CRC",
+        "value": 740.0
+    },
+    {
+        "currencyCode": "CUC",
+        "value": 1.20
+    },
+    {
+        "currencyCode": "CUP",
+        "value": 32.5
+    },
+    {
+        "currencyCode": "CVE",
+        "value":110.0
+    },
+    {
+        "currencyCode": "CZK",
+        "value":26.30
+    },
+    {
+        "currencyCode": "DJF",
+        "value":216.5
+    },
+    {
+        "currencyCode": "DKK",
+        "value": 7.5
+    },
+    {
+        "currencyCode": "DOP", 
+        "value": 70.0
+    },
+    {
+        "currencyCode": "DZD",
+        "value": 160.0
+    },
+    {
+        "currencyCode": "EGP",
+        "value": 19.077465
+    },
+    {
+        "currencyCode": "ERN",
+        "value": 18.270335
+    },
+    {
+        "currencyCode": "ETB",
+        "value": 47.254779
+    },
+    {
+        
+        "currencyCode": "EUR",
+        "value": 1
+    },
+    {
+        "currencyCode": "FJD",
+        "value": 2.515023
+    },
+    {
+        "currencyCode": "FKP",
+        "value": 0.909639
+    },
+    {
+        "currencyCode": "GBP",
+        "value": 0.909479
+    },
```

Comment:
Currency exchange rates always change. How do we make sure we adapt these static file in the frontend over time (after a few month or years)?

Reply:
I had the same concern but @sagararyal said the fluctuations are not much. we can round off them  nearest number

Reply:
To name an example Thai Baht had a fluctuation between 34 up to 37 Baht for 1 € last year. 

![B5DC795B-5996-452E-B672-8C4EFD1CAC6C](https://user-images.githubusercontent.com/1532418/103382044-dd927680-4aed-11eb-94cc-9e331e9747e2.jpeg)

Reply:
@norbertschuler this should be fine, as we are requiring min amount as 2€ equivalent. stripe min is around 1 € or less so these fluctuations do not affect transactions.